src/nnet3/nnet-cudnn-simple-component.cc

```diff
@@ -294,6 +289,27 @@ void CuDNN3DConvolutionComponent::InitFromConfig(ConfigLine *cfl) {
   ok = ok && cfl->GetValue("filt-y-step", &filt_y_stride);
   ok = ok && cfl->GetValue("filt-z-step", &filt_z_stride);
 
+  // optional argument
```

Comment:
Okay. Sounds good. It would be useful if you said that this was done specifically for compatibility with the existing component.